modules/coo-versus-default-ocp-monitoring.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+
+// * observability/cluster_observability_operator/cluster-observability-operator-overview.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="coo-versus-default-ocp-monitoring_{context}"]
+= {coo-short} compared to default monitoring stack
+
+The {coo-short} components function independently of the default in-cluster monitoring stack, which is deployed and managed by the {cmo-first}.
+Monitoring stacks deployed by the two Operators do not conflict. You can use a {coo-short} monitoring stack in addition to the default platform monitoring components deployed by the {cmo-short}.
+
+The key differences between {coo-short} and the default in-cluster monitoring stack are shown in the table:
+
+[cols="1,3,3", options="header"]
+|===
+| Feature      | {coo-short}      | Default monitoring stack
+
+| **Scope and integration**
+| Offers comprehensive monitoring and analytics for enterprise-level needs, covering cluster and workload performance. 
+
+However, it lacks direct integration with {product-title} and typically requires an external Grafana instance for dashboards.
+| Limited to core components within the cluster, for example, API server and etcd, and to OpenShift-specific namespaces. 
+
+There is deep integration into {product-title} including console dashboards and alert management in the console.
+
+| **Configuration and customization**
+| Broader configuration options including data retention periods, storage methods, and collected data types. 
+
+The {coo-short} can delegate ownership of single configurable fields in custom resources to users by using Server-Side Apply (SSA), which enhances customization.
+| Built-in configurations with limited customization options. You can set alerting and notification methods but you cannot adjust storage or retention policies.
```

Comment:
The above config description is not correct, according to the following doc, CMO can config retention time and persistent storage.
https://86074--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/monitoring/configuring-the-monitoring-stack#modifying-retention-time-and-size-for-prometheus-metrics-data_configuring-the-monitoring-stack
https://86074--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/monitoring/configuring-the-monitoring-stack#configuring-persistent-storage_configuring-the-monitoring-stack

Reply:
This content is based on https://access.redhat.com/articles/7096720 which was published in Dec 2024 - I can check with the author

Reply:
> The default monitoring stack did support configure retention time and pv
>the default is 15day and we do not recommand them configure more than 15d
>it just COO provide more flexible retention time

Reply:
I am going to leave in 

> Built-in configurations with limited customization options.

but I will remove the explicit mentions 

>You can set alerting and notification methods but you cannot adjust storage or retention policies.